src/test/java/dk/cngroup/trainings/spring/springassignment/controller/web/LoginControllerTest.java

```diff
@@ -47,30 +53,6 @@ public void nonLoggedUserRedirectionTest() throws Exception {
 				.andExpect(redirectedUrl("http://localhost/web/login"));
 	}
 
-	@Test
-	public void loggedUserRedirectionTest() throws Exception {
```

Comment:
why are you no longer testing this?

Reply:
I will re-implement tests again for LoginController